packages/ui-react/src/components/TextField/TextField.tsx

```diff
@@ -16,6 +16,7 @@ type InputOrTextAreaProps =
   | ({ multiline: true; inputRef?: never; textAreaRef?: RefObject<HTMLTextAreaElement> } & TextAreaProps);
 
 type Props = {
+  name: string;
```

Comment:
To make this more complete, let's destruct name from the props and change L45 to:
`const id = useOpaqueId('text_field', name);`

(I also changed the dash to an underscore, because this is preferred in id's)

Reply:
I agree. But it doesn't work for no apparent reason. When you're back lets look at it together!

Reply:
Fixed it!

Reply:
We fixed it from the huddle